From 5c9e2aeb0e78b5bbefa4be8d77f2a0bbfa1ac016 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Leonard=20Janis=20Robert=20K=C3=B6nig?= <ljrk@ljrk.org>
Date: Wed, 9 Mar 2022 09:37:12 +0000
Subject: Fix metainfo not validating

Validate with

    flatpak run org.freedesktop.appstream-glib validate ch.threema.threema-web-desktop.appdata.xml
---
 ch.threema.threema-web-desktop.metainfo.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ch.threema.threema-web-desktop.metainfo.xml b/ch.threema.threema-web-desktop.metainfo.xml
index 06b81b5..10e8d44 100644
--- a/ch.threema.threema-web-desktop.metainfo.xml
+++ b/ch.threema.threema-web-desktop.metainfo.xml
@@ -15,11 +15,11 @@
   </description>
   <screenshots>
     <screenshot>
-      <caption>The Threema for desktop main window.</caption>
+      <caption>The Threema for desktop main window</caption>
       <image type="source">https://raw.githubusercontent.com/threema-ch/threema-web-electron/develop/assets/threema-desktop-screenshot1.png</image>
     </screenshot>
     <screenshot>
-      <caption>The Threema for desktop main window.</caption>
+      <caption>The Threema for desktop main window</caption>
       <image type="source">https://raw.githubusercontent.com/threema-ch/threema-web-electron/develop/assets/threema-desktop-screenshot2.png</image>
     </screenshot>
   </screenshots>
-- 
2.35.1

